packages/design-system/src/components/Tooltip/Tooltip.jsx

```diff
@@ -68,24 +73,40 @@ export class Tooltip extends React.Component {
   }
 
   setTooltipActive(active) {
-    this.setState({ active }, () => {
-      this.popper.forceUpdate();
-    });
+    if (active !== this.state.active) {
```

Comment:
Adds `onOpen` and `onClose` event handlers, also prevents unnecessary state updates